LondonBritishLibrary/orient/BLorient758.xml

```diff
@@ -55,57 +54,81 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is in a fine hand, of the latter part of the <date>17th century.</date>.
+                                <desc>The manuscript is in a fine hand.
                                 </desc>
-                                
-                                <date notBefore="1650" notAfter="1700" resp="PRS10171Wright"/>
-                                
+                                <date notBefore="1650" notAfter="1700" resp="PRS10171Wright">
+                                  Of the latter part of the 17th century</date>
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
-                            
+                            <list>
+                              <item xml:id="e1">
+                                <locus from="3r"></locus>
+                                <desc>The name of the first owner, on <locus target="#3r"></locus> has
+                                been erased; and that of the second has also been partially effaced, only
+                              the word <foreign xml:lang="gez">ክርስቶስ፡</foreign> being left.</desc>
+                              </item>
+                              <item xml:id="e2">
+                                <locus target="#146a"></locus>
+                                <desc>Pen trial by one <persName ref="PRS13035SahlaMikael"></persName>.</desc>
+                              </item>
+                            </list>
+                            <list>
+                              <item xml:id="a1">
+                                <locus target="#3r"></locus>
+                                <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                    <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                <q xml:lang="gez">ማር፡ ይስሐቅ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                        </q>
+                              </item>
+                              <item xml:id="a2">
+                                <desc type="Genealogy"><title type="complete" ref="LIT3143Ledata" xml:lang="gez">
+                                ልደተ፡ መነኮሳት፡</title>Genealogy of monks, beginning with
+                                <persName ref="PRS1962Anthony"></persName> and ending with <persName ref="PRS9162TaklaHa"></persName>.
+                              </desc>
+                              </item>
+                              <item xml:id="a3">
+                                <desc type="GuestText">The names of the doctors of <placeName ref="INS0341DL"></placeName>
```

Comment:
"doctors"? better maybe mamher or rather abbots (look like abbots to me)? or should we leave Wright's version?